.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Let me know if there's a reason we don't want to do this. When I tried running `yarn` from the root of the repo on Node 18 I was getting

```
[15:30:49] 'bundleTask' errored after 118 ms
[15:30:49] Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:71:19)
    at Object.createHash (node:crypto:133:10)
```

Pretty sure it's a webpack problem with the version we're on: https://github.com/webpack/webpack/issues/14532